Philips.CodeAnalysis.MaintainabilityAnalyzers/CopyrightPresentAnalyzer.cs

```diff
@@ -76,5 +82,25 @@ private static void CreateDiagnostic(SyntaxNodeAnalysisContext context, Location
 			Diagnostic diagnostic = Diagnostic.Create(Rule, location);
 			context.ReportDiagnostic(diagnostic);
 		}
+
+		private static bool CheckCopyrightStatement(SyntaxNodeAnalysisContext context, SyntaxTrivia trivia) {
+			var comment = trivia.ToFullString();
+			// Check the copyright mar itself
+			var hasCopyright = comment.Contains("©") || comment.Contains("Copyright");
+			
+			// Check the year
+			bool hasYear;
+			lock (yearRegex)
```

Comment:
You don't need this, Regex objects are thread-safe for just calling something like Match.

Reply:
You're right, removed the locks again.